src/XRFeitoriaUnreal/Content/Python/utils.py

```diff
@@ -118,8 +118,8 @@ def import_asset(path: Union[str, List[str]], dst_dir_in_engine: Optional[str] =
         asset_tools.import_asset_tasks(import_tasks)
         asset_paths.extend([path.split('.')[0] for path in import_task.get_editor_property('imported_object_paths')])
 
-    unreal.EditorAssetLibrary.save_directory(dst_dir_in_engine, False, True)  # save assets
-    unreal.log(f'Imported assets: {asset_paths}')
+        unreal.EditorAssetLibrary.save_directory(dst_dir, False, True)  # save assets
+        unreal.log(f'Imported asset: {p}')
```

Comment:
L88, `p` -> `path`, `path` -> `paths` or `path_list`

Reply:
fixed in [`e02c4e4` (#8)](https://github.com/openxrlab/xrfeitoria/pull/8/commits/e02c4e4fa3e5aec00688c56c24b86fe1b9dacbac)